### PR TITLE
Test setup

### PR DESCRIPTION
**Pull request by harshil1712**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+language: node_js
```
Comment (abhishek97) [inline .travis.yml]: Also need to run tests on travis

File: package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "description": "Generic Base Controller for express to generate JSON API response",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "mocha",
```
Comment (abhishek97) [inline package.json]: an example test also required

**Comment by harshil1712** [comment]
I have made the changes. The test runs for a PR. Here is the demo https://github.com/harshil1712/express-jsonapi-controller/pull/2

**Comment by abhishek97** [comment]
The issue is already closed.

**Pull request closed without merging**